services/apps/cron_service/src/jobs/nangoTrigger.job.ts

```diff
@@ -37,7 +37,7 @@ const job: IJobDefinition = {
 
       const platform = platformToNangoIntegration(int.platform as PlatformType, settings)
 
-      if (platform === NangoIntegration.GITHUB && !settings.nangoMapping) {
+      if (!platform || (platform === NangoIntegration.GITHUB && !settings.nangoMapping)) {
```

Comment:
Here I would say that perhaps this new platform needs to be added to the `platformToNangoIntegration` and the "reverse" function `nangoIntegrationToPlatform`. And this won't be undefined then right?

Reply:
Oh it was added - hm - how come platform is undefined then?